### DISCUSSION TITLE
Stuck at 99.9%,  #29. Suffocate

### DISCUSSION THREAD

**Discussion by kokonutistaken** [cssbattle/forum#325] [Q&A]
The image looks exactly the same but it's not a 100%

Code:

<div class="star"></div>
<div class="cir">
  <span></span>
  <span></span>
  <span></span>
  <span></span>
</div>

<style>
  body{
    margin:0;
    padding:0;
    display:flex;
    justify-content:center;
    align-items:center;
    height:100%;
    width:100%;
    background:#F3AC3C;
  }
  .star {
    position:absolute;
    width: 140px;
    height: 140px;
    background: #1A4341;
    transform:rotate(45deg);
  }
  .cir{
    height:400px;
    width:400px;
    background:transparent;
    font-size:0;
    z-index:1;
  }
  span{
    display:inline-block;
    height:200px;
    width:200px;
    background:#F3AC3C;
    border-radius:50%;
  }
</style>

**Comment by Beowolve** [comment]
You should post the full code and warp it in a code element, otherwise we can't help you.

**Comment by joewinbpaul** [comment]
I also am stuck <p> No matter what method I try it always stays at 99.9% even tho the images are literally identical <p> I even checked the code in Beo's previewer but I still can't understand why it is 99.9% when its literally 100% identical <p> <p> Here is my code: <p> `<style>*{background:radial-gradient(1q,#1a4341,99q,#f3ac3c 0)}*>*{margin:-50 200 150 0;background:#f3ac3c;border-radius:50%;box-shadow:0 211.5q#f3ac3c;-webkit-box-reflect:right` <p> Can u tell what is wrong here?

**Reply by Gabb192** [reply]
This is one of the hardest target to achieve 100%, especially with box-shadow, I suggest you to try and try again until you get it.

**Reply by joewinbpaul** [reply]
> This is one of the hardest target to achieve 100%, especially with box-shadow, I suggest you to try and try again until you get it.

...... ok? so u mean that using box-shadow is just making it harder to achieve right?

**Reply by joewinbpaul** [reply]
> This is one of the hardest target to achieve 100%, especially with box-shadow, I suggest you to try and try again until you get it.

By any chance is target 63 also one of those targets in which u need to be pixel-perfect? Cuz I am not getting a 100% match even after using decimals in the code <p> This is my code: <p> `<p><p><style>body{background:#191919;border:11q solid;color:5DBCF9;height:49;margin:114 164 146 166}p{height:70;width:70;border-radius:50%50%0;box-shadow:inset 0 0 0 10px;translate:-70px -70px;-webkit-box-reflect:right 53q}p+p{transform:scaleY(-1);translate:-70px -36px` <p> If u check the difference the maximum u can see are some _very thin ragged_ lines. But in other targets u can easily get 100% even with _ragged lines_ bigger/longer than that

**Reply by joewinbpaul** [reply]
I removed the decimals in the code as they are not making any difference either

**Comment by Salehhs** [comment]
<div class="T"></div>
<div class="R"></div>
<div class="L"></div>
<style>
  body{
    background:#62306D;
     position:vertical;
  }
  div {
        background: #F7EC7D;
     }
  .T{
   border-top-left-radius: 80px;
   border-top-right-radius: 80px;
   width: 160px;
   height: 80px;
    position: absolute;
    top:70px;
    left: 120px;
   }
  .L{
    border-bottom-left-radius: 50% ;
    width: 80px;
    height: 80px;
    position: absolute;
    bottom:70px;
    right:120px;
    background:#F7EC7D;
    }
  .r{
    border-bottom-right-radius:50% ;
    width: 80px;
    height: 80px;
    position: absolute;
    bottom:70px;
    left:120px;
    }
  </style>

**Reply by joewinbpaul** [reply]
> <style> body{ background:#62306D; position:vertical; } div { background: #F7EC7D; } .T{ border-top-left-radius: 80px; border-top-right-radius: 80px; width: 160px; height: 80px; position: absolute; top:70px; left: 120px; } .L{ border-bottom-left-radius: 50% ; width: 80px; height: 80px; position: absolute; bottom:70px; right:120px; background:#F7EC7D; } .r{ border-bottom-right-radius:50% ; width: 80px; height: 80px; position: absolute; bottom:70px; left:120px; } </style>

What is this??


Oh wait this is just a long solution for target #134...

**Comment by fredowashere** [comment]
```
<div f></div><div r></div><div e></div><div d></div>
<style>
  body {
    margin: 0;
    display: grid;
    grid-template: 1fr 1fr / 1fr 1fr;
    background: #1a4341;
  }
  div {
    background: #f3ac3c;
  }
  [f] {
    border-radius: 0 0 100px 0;
  }
  [r] {
    border-radius: 0 0 0 100px;
  }
  [e] {
    border-radius: 0 100px 0 0;
  }
  [d] {
    border-radius: 100px 0 0 0;
  }
```
100%

**Reply by Beowolve** [reply]
If you want do it that way, you can simplify it to this for example:

`<dt><dt><dt><dt><style>body{margin:-100;display:grid;grid-template:1fr 1fr/1fr 1fr;background:#1a4341}*{background:#f3ac3c;border-radius:25vw`

Think "out of the box" ;)

**Reply by fredowashere** [reply]
> If you want do it that way, you can simplify it to this for example:
> 
> `<dt><dt><dt><dt><style>body{margin:-100;display:grid;grid-template:1fr 1fr/1fr 1fr;background:#1a4341}*{background:#f3ac3c;border-radius:25vw`
> 
> Think "out of the box" ;)

WOW, how cool is that negative margin is sort of scaling up the entire thing? Never thought about it that way. To be honest I didn't even think it was possible. Does it work on all instances or is this exploiting the box-sizing thing?
